### PR TITLE
graphql-tools/scalars: copy editing

### PR DESCRIPTION
**Pull request by glasser**
A light pass of grammar and typo fixes.

Also make sure to consistently use American "serialiZation"
spelling (consistent with the API name) and long lines for
paragraphs (as arbitrary linebreaks do show up in the HTML).

### CODE REVIEW

**Comment by glasser** [comment]
By the way, while I'm looking at this file: I think there's a good opportunity to do better documentation of the actual 3 scalar APIs here.  The original GraphQL docs don't really explain what the difference between parseValue and parseLiteral is, why you need both, etc.  And I'm guessing from the validation section that returning `null` from parseValue/parseLiteral causes some kind of error, but that's only a guess from context?  I can't help with these issues because I need to read the docs I'm imagining first :)

**Comment by glasser** [comment]
Hmm, another thing possibly worth adding to this file (in another PR?) — a note that apollo-client doesn't particularly support this and you have to basically do unrelated manual conversion on the client.

**Comment by glasser** [comment]
I added a note about custom scalars to this PR (and linked to https://github.com/apollostack/apollo-client/issues/585)

File: source/graphql-tools/scalars.md [hunk 1]
```diff
@@ -6,32 +6,23 @@ description: Add custom scalars to a GraphQL schema.
...
 ```js
 scalar MyCustomScalar
 ```
 
-Consequently, you need to define the behaviour of the scalar, the way it will be serialised
-when the value is sent to the client, or how the value is resolved when coming from the client.
-For this purpose, each scalar needs to define three methods: `serialize`, `parseValue` and `parseLiteral`.
-In Apollo, these functions are defined with the `__` prefix: `__serialize`, `__parseValue` and `__parseLiteral`.
-Let's look at couple examples to demonstrate the potential of custom scalars.
+Consequently, you need to define the behaviour of the scalar: how it will be serialized when the value is sent to the client, and how the value is resolved when received from the client. For this purpose, each scalar needs to define three methods: `serialize`, `parseValue` and `parseLiteral`. In `graphql-js`, these functions are defined with the `__` prefix: `__serialize`, `__parseValue` and `__parseLiteral`. (Note that [Apollo Client does not currently support custom scalars](https://github.com/apollostack/apollo-client/issues/585), so there's no way to automatically apply the same transformations on the client side.)
```
Comment (helfer) [inline source/graphql-tools/scalars.md]: Actually, the `__` is specific to graphql-tools. In graphql-js you don't need them.

**Comment by helfer** [comment]
Just one comment, other than that it looks good, thanks!

**Comment by glasser** [comment]
I added a commit to clarify the double underscore.

**Comment by helfer** [comment]
Thanks, looks great!

**Pull request merged**